### PR TITLE
Allow In-line Strings for Request Mapping Template

### PR DESCRIPTION
**Pull request by jzeron**
This PR is to ad inline strings for request mapping templates as an option in addition to template paths.  The string must begin with "VTL:"

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -836,14 +836,52 @@ class ServerlessAppsyncPlugin {
...
+        let responseMappingTemplate = fs.readFileSync(respTemplPath, 'utf8');
+        return responseMappingTemplate
+      }
+    }
+    else { // return template at default path: /mapping-templates/`${type}`/`${field}`/response.vtl
+      let respTemplPath = path.join(config.mappingTemplatesLocation, `${type}.${field}.response.vtl`);
+      let responseMappingTemplate = fs.readFileSync(respTemplPath, 'utf8');
+      return responseMappingTemplate;
+    }
+  }
```
Comment (bboure) [inline index.js]: @jzeron Can we make these two methods `DRY`er?

Comment (jzeron) [reply index.js]: Open to suggestions to make lighter...  We are on holiday this week, but can revisit soon to address unit tests & docs...

**Comment by bboure** [comment]
Hey @jzeron any progress on this?
Do you want me to take over?

**Comment by bboure** [comment]
This got a bit old and outdated. Will close this for now
@jzeron let me know if that's something you'd like to follow up

**Pull request closed without merging**